build.rs

```diff
@@ -38,20 +39,12 @@ fn main() {
     let _ = fs::remove_file(dst.join("build/CMakeCache.txt"));
 
     #[cfg(windows)] {
-        if dst.join("lib/pkgconfig").join("aom.pc").exists() {
-            env::set_var("PKG_CONFIG_PATH", dst.join("lib/pkgconfig"));
-            pkg_config::Config::new().statik(true).probe("aom").unwrap();
-        } else { // MSVC
-            let bin_dir = if debug {
-                "Debug"
-            } else {
-                "Release"
-            };
-            println!("cargo:rustc-link-search=native={}", dst.join("build").join(bin_dir).to_str().unwrap());
-            println!("cargo:rustc-link-lib=static=aom");
-        }
+        println!("cargo:rustc-link-search=native={}", dst.join("build").to_str().unwrap());
+        println!("cargo:rustc-link-search=native={}", dst.join("build/Debug").to_str().unwrap());
```

Comment:
It is fine even if only one of the two will exist and have content (since it is built on the release o debug path for cargo already) and rustc does not seem to complain about non-existing paths.